firebaseMsgSwGenerator.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Can the name more closely match the template and generated response? Maybe `firebase-messaging-sw-generator.js`? Another option would be to go more generic, so that it works on any similar `.tmpl` file.